.stackblitz/codeflow.json

```diff
@@ -4,7 +4,7 @@
       "enabled": false
```

Comment:
If we're going to disable everything, should we remove the file entirely?

Reply:
Their documentation mentions disabling the bot via the configuration file or through admin permissions for the organization: https://developer.stackblitz.com/codeflow/integrating-codeflowapp-bot#disabling-codeflowapp-bot

My thoughts were disabling it via the configuration and then follow-up with deleting the config file to make sure it is in fact disabled for the repository. 

I believe Codeflow is closed sourced, so I'm unable to see how it detects the enabled status (if it reads from `main`, stashes to an external service, etc.).